common/src/main/java/org/conscrypt/ConscryptCertStore.java

```diff
@@ -0,0 +1,25 @@
+package org.conscrypt;
+
+import java.security.cert.X509Certificate;
+import java.util.Set;
+
+/**
+ * A certificate store that supports additional operations that are used in
+ * TrustManagerImpl.  This is primarily implemented by the cert store on the
+ * Android platform.
+ */
+@Internal
```

Comment:
Can this be not Internal/package private? Currently the Network Security Config subclasses TrustedCertificateStore to provide the certstore, which is kinda of ugly

Reply:
Sure, it can be public.  I'll keep it marked `@Internal`, since that's just a documentation annotation to indicate that it might change out from under people, but that's not a problem for Network Security Config.